i6_models/parts/rasr_fsa.py

```diff
@@ -139,7 +235,16 @@ def build_single(self, seq_tag: str) -> Tuple[int, int, np.ndarray, np.ndarray]:
         raw_fsa = self.builder.build_by_segment_name(seq_tag)
         return raw_fsa
 
-    def build_batch(self, seq_tags: Iterable[str]) -> WeightedFsa:
+    def _append_fsa(self, a, b):
```

Comment:
Typing?

```suggestion
    def _append_fsa(self, a: WeightedFsa, b: WeightedFsa) -> WeightedFsa:
```

Also can be static? Or better move out of the class?